agent/consul/leader.go

```diff
@@ -344,6 +344,8 @@ func (s *Server) revokeLeadership() {
 
 	s.stopConfigReplication()
 
+	s.stopACLReplication()
```

Comment:
Calling attention to this. Seems like this has always been missing? I noticed there were no longer any calls to `stopACLReplication` after the last PR merged, and it seems like it should be called here.